src/main/resources/wildfly/configure-wildfly.cli

```diff
@@ -30,3 +30,7 @@ data-source add \
 # so use with care and only enable this for debugging purposes.
 #/subsystem=logging/root-logger=ROOT:write-attribute(name=level,value=DEBUG)
 #/subsystem=logging/console-handler=CONSOLE:write-attribute(name=level,value=DEBUG)
+
+# This sets the max post size accepted by the server/RESTEasy.
+# NOTE: This is higher than our limit of 80MB because there seems to be some overhead.
+/subsystem=microprofile-config-smallrye/config-source=props:add(properties={"dev.resteasy.entity.file.threshold" = "100MB"})
```

Comment:
Hard-coded op 100MB? Kan het niet via een ENV var?

Reply:
Nee dat kan niet. Dit is ook een env var maar een compleet andere orde dan die env vars die we gebruiken om ons ZAC Docker Image te bouwen. Deze 'env var' of property hier is voor het configureren van WildFly. Dat gaat via Gradle, via Maven en vervolgens via deze file. Het doorgeven in die hele keten van een Gradle env var is denk ik niet te doen..

Reply:
Maar dit is inderdaad wel een ding. tenzij we ook deze waarde uit een env var kunnen halen heeft die MAX_FILE_SIZE_MB env var geen enkele zin in feite. Die kunnen we dan veel beter echt hardcoded op 80MB zetten en helemaal niet meer als env var doen. Ik denk dat dat het beste is.

Reply:
Juist, dat begrijp ik dan beter. Ik denk dat het trouwens wel mogelijk is om Wildfly bij het opstarten env vars te laten lezen om daarmee de runtime configuratie te zetten. Het lijkt me sterk dat je dit moet hard-coden in de cli. Maar goed, ik ben niet de expert.

Reply:
Dus actie = constante maken en geen var hiervoor gebruiken !

Reply:
Denk dat je gelijk hebt hoor @bas-info-nl maar dat zouden we dan moeten uitzoeken. Ik denk voor het beste om hiervan een constante te maken. Ook al zou je helemaal aanpasbaar kunnen maken dan nog zit er altijd een grens aan. Heeft denk ik weinig zin. Je wilt helemaal niet dat Dimpact of zelfs een gemeente dit aan kunnen passen wat mij betreft.